imports/server/users.ts

```diff
@@ -29,6 +29,31 @@ Meteor.publish('huntMembers', function (huntId: string) {
   return MeteorUsers.find({ hunts: huntId }, { fields: { hunts: 1 } });
 });
 
+Meteor.publish('displayNames', function () {
+  if (!this.userId) {
+    return [];
+  }
+
+  return MeteorUsers.find({}, { fields: { 'profile.displayName': 1 } });
+});
+
+Meteor.publish('avatars', function () {
+  if (!this.userId) {
+    return [];
+  }
+
+  return MeteorUsers.find({}, { fields: { 'profile.discordAccount': 1 } });
+});
+
+Meteor.publish('profiles', function () {
+  if (!this.userId) {
+    return [];
+  }
+
+  // For now, all user profiles are public, including email address
+  return MeteorUsers.find({}, { fields: { 'emails.address': 1, profile: 1 } });
+});
+
```

Comment:
Given the sensitivity around the users model, I decided that life would probably be easier if we had a more more-limited publishes rather than the singular "pass your own query" funtimes from the old `mongo.profiles` publish. (I'm not yet proposing an official style change yet, but in general I'm a little over the "pass your own query" style of publishes.). Long-term, my plan is to make these take a `huntId` parameter and then do some :sparkles:query and projection magic:sparkles:, but reserving that for future work. For now, this just preserves exactly the same functionality as we had before.